extensions/fairifier-rdf-extension/dtls/fairifier/GetFairDataPointInfoCommand.java

```diff
@@ -1,129 +1,160 @@
+
 package org.dtls.fairifier;
 
-import java.lang.Exception;
-import java.lang.System;
-import java.io.InputStreamReader;
 import java.io.BufferedReader;
 import java.io.IOException;
-import org.deri.grefine.rdf.utils.HttpUtils;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import org.json.JSONWriter;
-import org.json.JSONException;
-import com.google.refine.commands.Command;
-import org.eclipse.rdf4j.rio.Rio;
-import org.eclipse.rdf4j.rio.RDFParseException;
-import org.eclipse.rdf4j.rio.RDFHandlerException;
+
+import org.apache.http.HttpException;
+import org.apache.http.HttpResponse;
+import org.deri.grefine.rdf.utils.HttpUtils;
+import org.eclipse.rdf4j.model.IRI;
+import org.eclipse.rdf4j.model.impl.SimpleValueFactory;
 import org.eclipse.rdf4j.rio.RDFFormat;
-import org.eclipse.rdf4j.rio.RDFParser;
-import org.eclipse.rdf4j.rio.turtle.TurtleParser;
+import org.eclipse.rdf4j.rio.RDFHandlerException;
+import org.eclipse.rdf4j.rio.RDFParseException;
 import org.eclipse.rdf4j.rio.helpers.StatementCollector;
-import java.io.ByteArrayInputStream;
-import java.util.ArrayList;
-import java.io.InputStream;
-import nl.dtl.fairmetadata4j.io.*;
-import nl.dtl.fairmetadata4j.model.*;
-import java.util.List;
-import nl.dtl.fairmetadata4j.utils.*;
-import org.eclipse.rdf4j.model.impl.SimpleValueFactory;
-import org.eclipse.rdf4j.model.IRI;
+import org.eclipse.rdf4j.rio.turtle.TurtleParser;
+import org.json.JSONException;
+import org.json.JSONWriter;
+
+import com.google.common.net.HttpHeaders;
+
+import com.google.refine.commands.Command;
+
+import nl.dtl.fairmetadata4j.io.CatalogMetadataParser;
+import nl.dtl.fairmetadata4j.io.DatasetMetadataParser;
+import nl.dtl.fairmetadata4j.io.FDPMetadataParser;
+import nl.dtl.fairmetadata4j.model.CatalogMetadata;
+import nl.dtl.fairmetadata4j.model.DatasetMetadata;
+import nl.dtl.fairmetadata4j.utils.MetadataParserUtils;
+
 /**
  * A command to get all the metadata from a FAIR Data Point(FDP)
+ * 
  * @author Shamanou van Leeuwen
  * @date 1-11-2016
  *
  */
 
-public class GetFairDataPointInfoCommand extends Command{
+public class GetFairDataPointInfoCommand extends Command {
+
     private SimpleValueFactory f;
```

Comment:
Not something from this commit, but this can be a constant. Right now it's initialized every time `doPost` is invoked.